py/trtorch/csrc/trtorch_py.cpp

```diff
@@ -9,12 +9,106 @@
 #include "torch/custom_class.h"
 #include "torch/script.h"
 #include "torch/torch.h"
-
+#include "utils.h"
+using namespace nvinfer1;
+using namespace pybind11::literals;
 namespace py = pybind11;
 
 namespace trtorch {
 namespace pyapi {
 
+template <typename Derived>
+class pyCalibratorTrampoline : public Derived {
+ public:
+  using Derived::Derived; // Inherit constructors
+
+  int getBatchSize() const noexcept override {
+    PYBIND11_OVERLOAD_PURE_NAME(int, Derived, "get_batch_size", getBatchSize);
+  }
+
+  bool getBatch(void* bindings[], const char* names[], int nbBindings) noexcept override {
+    py::gil_scoped_acquire gil{};
+
+    py::function pyGetBatch = tensorrt::utils::getOverload(static_cast<Derived*>(this), "get_batch");
```

Comment:
What is the tensorrt namespace?

Reply:
Fixed it. This was inherited from TRT source code. I cleaned it up to be as minimal and relevant as possible.